test/inputs/gdb.js

```diff
@@ -38,4 +38,14 @@ describe('from(gdb)', () => {
       }
     })
   })
+  it('should end stream early', async () => {
```

Comment:
This test does not actually test that the stream closes, it just increments a number, checks that number, and closes a stream.